meta-ros1-melodic/conf/ros-distro/include/melodic/generated/superflore-ros-distro.inc

```diff
@@ -6172,7 +6172,7 @@ ROS_UNRESOLVED_PLATFORM_PKG_ftdi-eeprom = "UNRESOLVED-ftdi-eeprom"
 ROS_UNRESOLVED_PLATFORM_PKG_gawk = "UNRESOLVED-gawk"
 ROS_UNRESOLVED_PLATFORM_PKG_gazebo = "UNRESOLVED-gazebo"
 ROS_UNRESOLVED_PLATFORM_PKG_gdal-bin = "UNRESOLVED-gdal-bin"
-ROS_UNRESOLVED_PLATFORM_PKG_geographiclib-tools = "UNRESOLVED-geographiclib-tools"
+ROS_UNRESOLVED_PLATFORM_PKG_geographiclib-tools = "geographiclib-tools"
```

Comment:
This is generated file, don't edit it.

You can add
ROS_UNRESOLVED_PLATFORM_PKG_geographiclib = "geographiclib"
for the build time dependencies and
ROS_UNRESOLVED_PLATFORM_PKG_geographiclib-tools = "geographiclib-tools"
for runtime dependencies in ros-distro.inc.
But as in many cases the package.xml doesn't distinguish between these 2 very well.

Reply:
I edited it as a temporary fix until everything is generated again, will move to ros-distro.inc.